### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by stephenjudkins** [brendanhay/amazonka#1001]
There have been a good amount of changes since since `2.0.0`. There is at least one we'd like to use. Is there anything I could do to help shepherd through a new release?

**Comment by tysonzero** [comment]
Seconding this, I need the fixes in #973 to be able to build this library with the ghc version I am using (9.8.2).

**Comment by ysangkok** [comment]
These are the issues that need to be resolved for 2.1 to get released: https://github.com/brendanhay/amazonka/milestone/4

**Comment by endgame** [comment]
I am currently studying for an AWS re-qual and that's eating all of my free time. Once that's done, I hope to find some things to push back to 2.2 and start working towards getting the important things finished.

**Comment by endgame** [comment]
#1006 would be relatively easy to pick up, I think, if someone wants to give it a crack.

**Comment by Bodigrim** [comment]
Would it be possible to have any release, compatible with GHC 9.8, in upcoming weeks please? It would be problematic if `amazonka` falls out of Stackage LTS again, and I'd expect Stackage to switch to GHC 9.8 relatively soon.

**Comment by arybczak** [comment]
Indeed, can we get a 2.0.1 release which is basically 2.0 + GHC 9.8 compatibility (and maybe other bugfixes)?

**Comment by ysangkok** [comment]
@arybczak You'd need `allow-newer` too, unless you're somehow using an older index-state (or you set the revision somehow, which I don't think cabal-install allows?) Because amazonka-core was revised to disallow GHC 9.8: https://hackage.haskell.org/package/amazonka-core-2.0/revisions/

**Comment by 9999years** [comment]
Yes, we also need #972 for GHC 9.8/9.10.

**Comment by ysangkok** [comment]
@endgame What are the easiest issues that need to get done for 2.1?

**Comment by endgame** [comment]
@ysangkok #1027 would be a good warm-up. #1017 would be a good PR to use as inspiration. If you're keen for more after that, I'll see if I can recommend something else. Are you more comfortable with the idea of chasing laziness bugs or reading other SDKs?

**Comment by juhp** [comment]
I assume the question was more 9.8/9.10 when or how to get there? :-)

**Comment by arybczak** [comment]
Is there anything particularly time consuming about creating a branch for 2.0.1 that's 2.0 + compatibility fixes for newer GHC versions? It's been over a year.
